client/src/components/registration-login/Login.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState, useContext} from 'react';
+import Button from 'react-bootstrap/Button';
+import Image from 'react-bootstrap/Image';
+import {Route, BrowserRouter as Router, Switch, Link } from 'react-router-dom';
+import AppContext from './../AppContext.js';
+import axios from 'axios';
+
+const Login = (props) => {
+  const {username, website, bio, setUsername, setBio, setWebsite, serverUrl, isArtist, setIsArtist} = useContext(AppContext)
+
+  const handleAccountNameChange = (e) => {
+    setUsername(e.target.value);
+  }
+
+  const loginHandler = ()=> {
+    axios.get(`${serverUrl}/username?${username}`)
+    .then(()=>{props.history.push('/TestHome')})
+    .catch(()=>{alert('invalid username')})
```

Comment:
1. indentation for .then and .catch
2. spacing for the callbacks. () => {
3. semicolon after catch statement